packages/typescript/ai-openrouter/tests/tools-per-model-type-safety.test.ts

```
@@ -0,0 +1,65 @@
+/**
+ * Per-model type-safety tests for OpenRouter provider tools.
+ *
+ * Positive cases: each supported (model, tool) pair compiles cleanly.
+ * Negative cases: unsupported (model, tool) pairs produce a `@ts-expect-error`.
+ */
+import { describe, it, beforeAll } from 'vitest'
```

ESLint

[error] Member 'beforeAll' of the import declaration should be sorted alphabetically.

(sort-imports)